### PR TITLE
Adds banner to header.

### LINKED ISSUE #126: add "Beta" banner to extension UI
Komoroske brought this up during review.

To manage expectations, we need to point people to this repo and indicate things are still in development

### PR DESCRIPTION
**Pull request by paullewis**
Fixes #126

### CODE REVIEW

File: extension/app/styles/lighthouse.css [hunk 1]
```diff
@@ -54,6 +56,24 @@ header h2 {
...
+  height: 32px;
+  font-size: 14px;
+  background: linear-gradient(to bottom, #D0021B 0%, #B30419 100%);
+  color: #FFF;
+  font-weight: bold;
+  text-shadow: 0 1px 1px rgba(0,0,0,0.3);
+  line-height: 32px;
+  text-align: center;
+  width: 140px;
+  transform:  translate(36px, 20px) rotate(45deg);
```
Comment (brendankenny) [inline extension/app/styles/lighthouse.css]: extra space

**Comment by paulirish** [comment]
screenshot: 
image

**Comment by brendankenny** [comment]
fancy!

**Comment by paulirish** [comment]
lgtm

**Pull request merged**